cellprofiler/modules/loadimages.py

```diff
@@ -436,199 +463,241 @@ def example_path_fn():
-            metadata field by joining the two fieldname values together. For example,
-            if <i>&lt;WellRow&gt;</i> is "A" and <i>&lt;WellColumn&gt;</i> is "01", a field
-            <i>&lt;Well&gt;</i> will be "A01". This is useful if your well row and column names are
-            separated from each other in the filename, but you want to retain the standard
-            well nomenclature.</p>"""))
+        'Regular expression that finds metadata in the file name',
+        '^(?P.*)_(?P[A-P][0-9]{2})_s(?P[0-9])',
+                get_example_fn=example_file_fn, doc="""\
+*(Used only if you want to extract metadata from the file name)*
+
+The regular expression to extract the metadata from the file name is
+entered here. Note that this field is available whether you have
+selected *Text-Regular expressions* to load the files or not. Please see
+the general module help for more information on construction of a
+regular expression.
+
+Clicking the magnifying glass icon to the right will bring up a tool for
+checking the accuracy of your regular expression. The regular expression
+syntax can be used to name different parts of your expression. The
+syntax *(?P<fieldname>expr)* will extract whatever matches *expr* and
+assign it to the measurement,\ *fieldname* for the image.
+
+For instance, a researcher uses plate names composed of a string of
+letters and numbers, followed by an underscore, then the well,
+followed by another underscore, followed by an “s” and a digit
+representing the site taken within the well (e.g.,
+*TE12345_A05_s1.tif*). The following regular expression will capture
+the plate, well, and site in the fields “Plate”, “Well”, and “Site”:
+
++----------------------------------------------------------------+------------------------------------------------------------------+
+| ^(?P<Plate>.*)_(?P<Well>[A-P][0-9]{1,2})_s(?P<Site>[0-9])   |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| ^                                                              | Start only at beginning of the file name                         |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| (?P<Plate>                                                     | Name the captured field *Plate*                                  |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| .*                                                            | Capture as many characters as follow                             |
++----------------------------------------------------------------+------------------------------------------------------------------+
+| _                                                             | Discard the underbar separating plate from well                  |
```

Comment:
Please add 1 more space in the first column of this line to get the table to format correctly.